apiserver/facades/client/modelmanager/services.go

```diff
@@ -9,9 +9,9 @@ import (
 
 	"github.com/juju/description/v9"
 
-	"github.com/juju/juju/apiserver/common"
 	"github.com/juju/juju/apiserver/facade"
 	jujucloud "github.com/juju/juju/cloud"
+	"github.com/juju/juju/core/agentbinary"
 	"github.com/juju/juju/core/assumes"
```

Comment:
On the following services can we remove these functions that are not being used?

**AccessService**:
- `ReadUserAccessLevelForTarget`

**MachineService**:
- `EnsureDeadMachine`
- `InstanceID`

**CredentialService**:
- `InvalidateCredential`

**ModelDefaultsService**
- `ModelDefaultsProvider`

**ModelService**
- `GetModelUser`
- `GetModelUsers`
- `ControllerModel`